tests/e2e/default/sms/messages-breadcrumbs.wdio-spec.js

```diff
@@ -0,0 +1,98 @@
+const utils = require('../../../utils');
+const commonElements = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const userFactory = require('../../../factories/cht/users/users');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const messagesPo = require('../../../page-objects/default/sms/messages.wdio.page');
+
+describe('Tasks tab breadcrumbs', () => {
+  const places = placeFactory.generateHierarchy();
+  const clinic = places.get('clinic');
+  const health_center = places.get('health_center');
+  const district_hospital = places.get('district_hospital');
+  const contact = {
+    _id: 'fixture:user:user1',
+    name: 'OfflineUser',
+    phone: '+12068881234',
+    place: health_center._id,
+    type: 'person',
+    parent: {
+      _id: health_center._id,
+      parent: health_center.parent
+    },
+  };
+  const contact2 = {
+    _id: 'fixture:user:user2',
+    name: 'OnlineUser',
+    phone: '+12068881235',
+    place: district_hospital._id,
+    type: 'person',
+    parent: {
+      _id: district_hospital._id,
+    },
+  };
+  const offlineUser = userFactory.build({
+    username: 'offlineuser',
+    isOffline: true,
+    place: health_center._id,
+    contact: contact._id,
+  });
+  const onlineUser = userFactory.build({
+    username: 'onlineuser',
+    roles: [ 'program_officer' ],
+    place: district_hospital._id,
+    contact: contact2._id,
+  });
+  const patient = personFactory.build({
+    _id: 'patient1',
+    phone: '+14152223344',
+    name: 'patient1',
+    parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
+  });
+  const contactNameSelector = ' .sender .name';
```

Comment:
This selector could be better in the `page-object/default/sms/messages.wdio.page.js` file. Could you please move it there?

Reply:
yup